internal/pipe/changelog/changelog.go

```diff
@@ -121,6 +121,32 @@ func buildChangelog(ctx *context.Context) ([]string, error) {
 	entries := strings.Split(log, "\n")
 	entries = entries[0 : len(entries)-1]
 	entries, err = filterEntries(ctx, entries)
+
+	for index, entry := range entries {
+		//  ci: auto assign project to issues/prs
+		//feat: add support for checksum generation for extra files (#2406) - thanks to <author_name> !!
+		splitEntry := strings.Split(entry, " ")
+		lastElement := splitEntry[len(splitEntry)-1]
+		if strings.HasPrefix(lastElement, "(#") &&
+			strings.HasSuffix(lastElement, ")") {
+			fmt.Println(entry)
+			commitHash := strings.Split(entry, " ")[0]
+			fmt.Println(commitHash)
```

Comment:
I think we can remove these two `Println`s. Maybe an info log would be nice for `--debug` flag?